apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -216,8 +216,8 @@ export function Workspace({
                               <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">Reloading</div><div class="col col-9">Use your browser reload button on this tab at any time, or close and re-open the tab.</div></div></li>
                               <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">Rebooting</div> <div class="col col-9">The <span class="badge badge-outline badge-light"><i class="fas fa-sync" aria-hidden="true"></i> Reboot</span> button will restart the virtual machine. Your files will remain intact.</div></div></li>
                               <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">Resetting</div> <div class="col col-9">The <span class="badge badge-outline badge-light"><i class="fas fa-trash text-secondary" aria-hidden="true"></i> Reset</span> button will delete all of your file edits and revert the virtual machine to its original state.</div></div></li>
-                              <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">Saving</div>    <div class="col col-9">Your files are automatically saved.</div></div></li>
                               <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">Grading</div>   <div class="col col-9">Use the <span class="badge badge-outline badge-light">Save &amp; Grade</span> button on the PrairieLearn question page to submit your files for grading.</div></div></li>
+                              <li class="list-group-item p-2"><div class="row"><div class="col col-3 font-weight-bold d-flex align-items-center">What to Tell Students</div> <div class="col col-9"><ul><li> How to save/submit </li> <li> Advise on workspace-specific instructions, e.g. How to interupt/restart the kernel in Jupyter workspaces </li> <li> Set expectations about workspace start times (in the worst case, a workspace may take several minutes as new machines are started and as images are pulled </li> </ul> 
```

Comment:
This line in particular is targeted at instructors, while this help page is targeted at students. As per https://github.com/PrairieLearn/PrairieLearn/issues/9213#issuecomment-2035008870, please add this to the instructor documentation instead of here. The documentation is found in https://github.com/PrairieLearn/PrairieLearn/blob/master/docs/workspaces/index.md.

Note that the instructions sent by @nwalters512 were general guidelines, you will need to expand on that to make full sentences. In particular, these are expected to be a new section for the documentation page.

Reply:
Thank you for the feedback, we have removed the line from workspace.html.ts and added a "what to tell students" section to the instructor documentation.